resources/refund_test.go

```diff
@@ -41,3 +41,19 @@ func TestRefundCreate(t *testing.T) {
 	assert.Equal(t, err, nil)
 	utils.TestResponse(jsonByteArray, []byte(fixture), t)
 }
+
+func TestRefundEdit(t *testing.T) {
+	url := constants.REFUND_URL + "/" + TestPaymentID
```

Comment:
```suggestion
	url := constants.REFUND_URL + "/" + TestPaymentID
```

Define a separate constant for `TestRefundID`

Reply:
done